packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { FieldKey, EmptyKey } from "../../tree";
+import { IEditableForest, TreeNavigationResult, ITreeSubscriptionCursor } from "../../forest";
+import { TreeSchemaIdentifier } from "../../schema-stored";
+
+export const proxySymbol = Symbol.for("editable-tree");
+
+/**
+ * A dummy placeholder for a Proxy target.
```

Comment:
I think this comment is wrong. The target for your proxy is ObjectEditableTree (the target is what is wrapped by the proxy, not the interface the proxy implements).
```suggestion
 * A tree with can be traversed and edited.
 * TODO: support editing.
```